src/newrelic/app.go

```diff
@@ -244,6 +245,9 @@ func (info *AppInfo) ConnectPayloadInternal(pid int, util *utilization.Data) *Ra
 		utilCopy := *util
 		data.Util = &utilCopy
 		data.Util.Hostname = data.Host
+		if info.DockerId != "" {
```

Comment:
Does this handle if the value is nil?

Reply:
The value at this point should never be `nil` since `DockerId` is defined as a string type and string types in Go can't be `nil`.